db/schema.rb

```diff
@@ -19,8 +19,12 @@
     t.string   "name"
     t.text     "description"
     t.decimal  "price"
-    t.datetime "created_at",  null: false
-    t.datetime "updated_at",  null: false
+    t.datetime "created_at",        null: false
+    t.datetime "updated_at",        null: false
+    t.decimal  "net_price"
+    t.string   "catalogue_number"
+    t.boolean  "new_product"
+    t.string   "country_of_origin"
```

Comment:
Dodaj jeszcze dla `new_product` i `name` warunek `null: false`.
Możesz dodać nowy plik migracji który to dodaje. 
Wygoogluj. `How to add null false to migraration rails` :)